runbotics/runbotics-orchestrator-ui/src/views/process/AddProcess.tsx

```diff
@@ -100,16 +132,16 @@ const AddProcess = () => {
             <Button
                 color="primary"
                 variant="contained"
-                onClick={() => setShow(true)}
-                startIcon={(
+                onClick={() => setShowDialog(true)}
+                startIcon={
                     <SvgIcon fontSize="small">
                         <PlusIcon />
                     </SvgIcon>
-                )}
+                }
             >
                 {translate('Process.Add.ActionName')}
             </Button>
-            {show && <AddProcessDialog open={show} onClose={() => setShow(false)} onAdd={handleAdd} />}
+            {showDialog && <AddProcessDialog open={showDialog} onClose={() => setShowDialog(false)} onAdd={handleAdd} />}
```

Comment:
i know that it was already there but lets correct it
I mean that we decide about visibility of this dialog by open prop and `{showDialog && <AddProcessDialog...`

IMO use only open prop, check if it works correctly otherwise adjust logic in this component